src/generators.ts

```diff
@@ -10,4 +12,17 @@ export const generateSingleQueryMashup = (queryName: string, query: string): str
     shared #"${queryName}" = 
     ${query};`;
 
+export const generateMultipleQueryMashup = (loadedQuery: QueryInfo, queries: ConnectionOnlyQueryInfo[]): string => {
+    let mashup: string = generateSingleQueryMashup(loadedQuery.queryName!, loadedQuery.queryMashup!);
+    queries.forEach((query: ConnectionOnlyQueryInfo) => {
+        const queryName = query.queryName ? query.queryName : "Query" + queries.indexOf(query);
```

Comment:
I'm not sure it is possible, but IIUC the user can set some of the queries names right? so what if the name Query with the index is already in used? do you want to check it?